packages/react-reconciler/src/ReactFiberPerformanceTrack.js

```diff
@@ -114,6 +114,54 @@ export function logComponentEffect(
   }
 }
 
+export function logYieldTime(startTime: number, endTime: number): void {
+  if (supportsUserTiming) {
+    const yieldDuration = endTime - startTime;
+    if (yieldDuration < 1) {
+      // Skip sub-millisecond yields. This happens all the time and is not interesting.
+      return;
+    }
```

Comment:
Do these become >=1ms when CPU throttling is turned on?

Reply:
Maybe for the really slow ones. Depends on the overhead of yielding on the platform I guess.